src/main/java/org/kuali/kfs/module/purap/document/PaymentRequestDocument.java

```diff
@@ -748,13 +751,16 @@ public void doRouteStatusChange(final DocumentRouteStatusChange statusChangeEven
         if (isAutoApprovedIndicator()) {
             updateAndSaveAppDocStatus(PaymentRequestStatuses.APPDOC_AUTO_APPROVED);
             // DOCUMENT PROCESSED .. //KFSCNTRB-1207 - UMD - Muddu -- end
-        }
-        if (getDocumentHeader().getWorkflowDocument().isProcessed()) {
+        } else if (getDocumentHeader().getWorkflowDocument().isProcessed()) {
+          // Cornell Customization : KFSPTS-34015: Made above line "else if" to be logic prior to 202304-19 changes.
```

Comment:
Is the "else if" change above still needed? I thought we determined that we didn't need it to fix the problem.

Reply:
This is what Cathy executed in sandbox. Please refer to feature branch link in JIRA sub-task for comparison. 
Logic would not be exactly the same as what she executed if that change was made.

Reply:
In that case, I'm okay with this change remaining in place for now, and we can do a follow-up fix later if it causes auto-approve-related issues.